test-app/app/src/main/java/com/tns/RuntimeHelper.java

```diff
@@ -105,13 +105,16 @@ public static Runtime initRuntime(Application app)
 					logger.write("Error while getting current proxy thumb");
 				e.printStackTrace();
 			}
-			ThreadScheduler workThreadScheduler = new WorkThreadScheduler(new Handler(Looper.getMainLooper()));
+
+			ThreadScheduler workThreadScheduler = new WorkThreadScheduler();
```

Comment:
The object is not completely setup when we pass it on to the runtime. Can we agree to move it entirely inside Runtime.java? @NativeScript/android-runtime